src/lib.rs

```diff
@@ -315,6 +317,17 @@ pub fn parse_many<B: AsRef<[u8]>>(input: B) -> Result<Vec<Pem>> {
 ///
 /// # Example
 /// ```rust
+///  use pem::{RefPem, encode};
+///
+///  let pem = RefPem {
+///     tag: "FOO",
+///     contents: &[1, 2, 3, 4],
+///   };
+///   encode(&pem);
```

Comment:
This example fails because you don't have an `Into<RefPem<'p>>` implementation for a `&RefPem<'p>` itself. So does the other example you added for the `encode_config` function. That said, I am not sure that having an implementation to go from a `&RefPem` to a `RefPem` makes a whole lot of sense and it is going to be weird that one of the ways you call it is `encode(&pem)` and the other way is `encode(ref_pem)`.